README.md

```
@@ -168,8 +171,16 @@ yarn publish
 - Clone the repository to a local folder: `git clone git@github.com:alkem-io/excalidraw.git` and create a feature branch to store your work.
 - Follow the original Excalidraw instructions below to run and debug with the included test application - Just `yarn ; yarn start` should work.
 - To test/debug Excalidraw inside our client-web application:
-  - Execute `npm link` in the root of your cloned repository.
-  - Go to your client-web folder and execute: `npm link @alkemio/excalidraw --save`
+  - Run in the excalidraw-fork root folder:
+    ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)